Assets/Saves/sample.yaml

```diff
@@ -1,49 +1,90 @@
-# version of the yaml parser
-# needed when changes are made to the parsing
-version: 1.0
```

Comment:
Gibt es einen Grund, warum Version nicht mehr benötigt wird?

Reply:
Gibt es einen Grund, warum Version benötigt wird?

Die YAML Dateien werden eh nur in diesem Repo sein und gleichzeitig mit den anderen commits geupdated.